lib/customer.rb

```diff
@@ -0,0 +1,49 @@
+require "csv"
+require "pry"
+require "awesome_print"
+
+class Customer
+
+  attr_reader :id, :email, :address
```

Comment:
You refactored your read/writers in the order, but not here. NBD, but worth keeping consistent.